doc/api/errors.md

```diff
@@ -714,6 +714,15 @@ Used when a child process is being forked without specifying an IPC channel.
 Used when the main process is trying to read data from the child process's
 STDERR/STDOUT, and the data's length is longer than the `maxBuffer` option.
 
+<a id="ERR_CLOSED_MESSAGE_PORT"></a>
+### `ERR_CLOSED_MESSAGE_PORT`
+<!--
+added: v10.5.0
```

Comment:
Anyone know if we should somehow apply metadata to show that it was removed in 11.12.0 and will now be re-added? I'm guessing @aduh95?

Reply:
I couldn't find a precedent, I guess we should either put the whole history:
```suggestion
added: v10.5.0
changes:
  - version: 11.12.0
    pr-url: https://github.com/nodejs/node/pull/26487
    description: The error message was removed.
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/38510
    description: The error message was reintroduced.
```
either ignore the EOL versions and consider this as new:
```suggestion
added: REPLACEME
```

Reply:
Thank you!!!